packages/alt-ergo/alt-ergo.0.95.2/opam

```diff
@@ -1,24 +1,34 @@
+(******************************************************************************
+* Please, do not modify this file without contacting the maintainers
+* to discuss the reasons of the modification.
+* In particular, dev-repo is not provided because it's private.
+******************************************************************************)
```

Comment:
At least for 0.95.2, the `conf-autoconf` dependency replaced the `autoconf` depext. I'm not sure why the `conf-autoconf` dependency was added for the other versions.